1-js/13-modules/02-import-export/article.md

```diff
@@ -306,133 +305,152 @@ import func from '/path/to/func.js';
+Como as funcionalidade exportadas estão espalhadas pelo pacote, podemos importá-las em `auth / index.js` e exportá-las neles:
 
 ```js
 // 📁 auth/index.js
+
+// import login/logout e imediatamente exportá-los
 import {login, logout} from './helpers.js';
 export {login, logout};
 
+// importar default como User e exportá-lo
 import User from './user.js';
 export {User};
-
-import Github from './providers/github.js';
-export {Github};
 ...
 ```
 
-"Re-exporting" is just a shorter notation for that:
+Agora os usuários do nosso pacote podem  `import {login} from "auth/index.js"`.
+
+A sintaxe `export ... from ...` é apenas uma notação mais curta para essa import-export:
 
 ```js
 // 📁 auth/index.js
+// importar login/logout e imediatamente exportá-los
 export {login, logout} from './helpers.js';
-// or, to re-export all helpers, we could use:
-// export * from './helpers.js';
 
+// importar default como User e exportá-lo
 export {default as User} from './user.js';
-
-export {default as Github} from './providers/github.js';
 ...
 ```
 
-````warn header="Re-exporting default is tricky"
-Please note: `export User from './user.js'` won't work. It's actually a syntax error. To re-export the default export, we must mention it explicitly `{default as ...}`, like in the example above.
+### Reexportando o export default
+
+O export default precisa de um tratamento separado ao reexportar.
+
 
-Also, there's another oddity: `export * from './user.js'` re-exports only named exports, exluding the default one. Once again, we need to mention it explicitly.
+Vamos dizer que temos `user.js`, e gostaríamos de reexportar a classe `User`
 
-For instance, to re-export everything, two statements will be necessary:
 ```js
-export * from './module.js'; // to re-export named exports
-export {default} from './module.js'; // to re-export default
+// 📁 user.js
+export default class User {
+  // ...
+}
 ```
 
-The default should be mentioned explicitly only when re-exporting: `import * as obj` works fine. It imports the default export as `obj.default`. So there's a slight asymmetry between import and export constructs here.
-````
+1. `export User from './user.js'` não funcionará. O que pode dar errado? ... Mas isso é um erro de sintaxe!
 
-## Summary
+    Para reexportar o export default, nós temos que escrever `export {default as User}`, como no exemplo acima.
 
-There are following types of `export`:
+2. `export * from './user.js'` reexporta apenas os exports com nome, mas ignora o default.
 
-- Before declaration:
+    Se desejarmos reexportar tanto os export com nome e o export default, serão necessárias duas declarações:
+    ```js
+    export * from './user.js'; // para reexportar exports com nome
+    export {default} from './user.js'; // para reexportar o export default
+    ```
+
+Essas esquisitices de reexportar o export default são um dos motivos pelos quais alguns desenvolvedores não gostam deles.
```

Comment:
I liked the word "esquisitices" :) nice one!

Reply:
Tks for the feedback :)